settings.gradle

```diff
@@ -4,59 +4,68 @@ rootProject.name = 'zipkin'
 // Zipkin's mostly common library //
 ////////////////////////////////////
 // Not all projects depend on this!
-include 'zipkin-common',
```

Comment:
there was some weird thing once upon a time that forced us to use commas instead of multiple lines of include. Maybe it was intellij?

Probably what we want to do is make a list, and then condtionally add to the end of it. After that, "include" that list.